index.js

```diff
@@ -0,0 +1,126 @@
+import express from 'express'
+import {DescribeTableCommand, DynamoDBClient, ListTablesCommand} from "@aws-sdk/client-dynamodb";
+import {
+    DeleteCommand,
+    DynamoDBDocumentClient,
+    GetCommand,
+    PutCommand,
+    ScanCommand,
+    UpdateCommand
+} from "@aws-sdk/lib-dynamodb";
+
+const app = express();
+const client = new DynamoDBClient({ region: "eu-north-1" });
```

Comment:
No need to change but AWS config parameters should always come from env variables. In case you want to run this code in a different account where region is different, you will need to update the code.